redwood-widget/src/iosMain/kotlin/app/cash/redwood/widget/RedwoodUIKitView.kt

```diff
@@ -53,6 +52,29 @@ public open class RedwoodUIKitView(
   }
 }
 
+public class RootUiView : UIView(cValue { CGRectZero }) {
```

Comment:
I really don't want to make this public. I found it hard to appease all the Kotlin/Native constraints and making this non-public (i.e., disallowing an open class extending `UIView`, disallowing a class extending `UIView` and a Kotlin class).

Reply:
I made this `private` within `TreehouseUIView` again.